### PR TITLE
TRUNK-5069 Add tests to PatientServiceImpl

### PR DESCRIPTION
**Pull request by dilekamadushan**
https://issues.openmrs.org/browse/TRUNK-5069
Description of what I changed

  Added tests to PatientServiceImpl

Checklist: I completed these to help reviewers :)

[x] My pull request only contains ONE single commit

[x] My IDE is configured to follow the code style of this project.

[x] I have added tests to cover my changes. (If you refactored
existing code that was well tested you do not have to add tests)

[x] I ran mvn clean package right before creating this pull request and
added all formatting changes to my commit.

[x] All new and existing tests passed.

[x] My pull request is based on the latest changes of the master branch.

### CODE REVIEW

**Comment by dkayiwa** [comment]
Why do your ticket links always end with "?filter=-1"?

**Comment by dilekamadushan** [comment]
@dkayiwa I modified the Issue link

**Comment by dilekamadushan** [comment]
@wluyima Feel free to review this PR

File: api/src/test/java/org/openmrs/api/PatientServiceTest.java [hunk 1]
```diff
@@ -2751,6 +2751,14 @@ public void unvoidPatient_shouldUnvoidPerson() throws Exception {
 		Assert.assertFalse(patient.getPersonVoided());
 	}
 	
+	@Test
+	public void unvoidPatient_shouldReturnNullWhenPatientIsNull() throws Exception {
+		
+		PatientService patientService = Context.getPatientService();
```
Comment (teleivo) [inline api/src/test/java/org/openmrs/api/PatientServiceTest.java]: there is already a `patientService` available for you. no need to get it from the context

File: api/src/test/java/org/openmrs/api/PatientServiceTest.java [hunk 2]
```diff
@@ -2751,6 +2751,14 @@ public void unvoidPatient_shouldUnvoidPerson() throws Exception {
...
+		Patient unVoidedPatient = patientService.unvoidPatient(null);
+		Assert.assertNull(unVoidedPatient);
```
Comment (teleivo) [inline api/src/test/java/org/openmrs/api/PatientServiceTest.java]: you can rewrite the test as (staticly importing assertNull)

```java
public void unvoidPatient_shouldReturnNullWhenPatientIsNull() throws Exception {

  assertNull(patientService.unvoidPatient(null);
}
```

Comment (dilekamadushan) [reply api/src/test/java/org/openmrs/api/PatientServiceTest.java]: @teleivo I changed the code according to your comments. Feel free to review the code. Thanks

**Comment by dilekamadushan** [comment]
@teleivo  Thank you very much for merging :)

**Pull request merged**